### DISCUSSION TITLE
Flag for canvas and webGL fingerprint Android System Webview

### DISCUSSION THREAD

**Discussion by Jackychans** [bromite/bromite#1767] [Q&A]
Hi,
I'm using Android System Webview (Bromite). I went through browserleaks and was able to see canvas fingerprint shown differently every time reload page.
However, webGL fingerprint remains the same. Already read through documentation but still not find necessary flag for it or how to enable/disable other flags.

Please advise, thanks a lot

**Comment by Jackychans** [comment]
I believe the flag would be #disable-webgl. But how could I enable this flag without rebuilding everything from source ?

And if this flag is enable (meaning webgl disabled), is that webgl fingerprint field would show nothing (from browserleaks) ?

**Comment by Jackychans** [comment]
I've taken a closer look at patch Add-flag-to-disable-WebGL.patch.

In order to disable webgl by default, I updated value just like photo below

![Screenshot 2022-02-08 114227](https://user-images.githubusercontent.com/6682672/153235228-ec73e916-8ba5-4007-91ac-2bba8e094970.png)


However, once build completed, the webgl value still shown up in browserleaks page.

Please advise me how to make webview disabled by default, thanks a lot

**Comment by csagan5** [comment]
There is an open issue about this: https://github.com/bromite/bromite/issues/131
